arduino/OpenMRNLite.h

```diff
@@ -524,8 +537,11 @@ class OpenMRN : private Executable
     /// List of objects we need to call in each loop iteration.
     vector<Executable *> loopMembers_{{this}};
 
+#if !defined(OPENMRN_FEATURE_SINGLE_THREADED) && \
```

Comment:
1) 
consider adding an 
```
#else
   static constexpr const bool haveExecutorThread_ = false;
#endif
```
because this will make code optimized away.

2)
it would be beneficial to not have to repeat the same complex `#ifdef` in multiple places in the code (3 currently!). This even overrides my desire to have certain ordering of methods vs class variables. Consider moving this variable declaration up into the other ifdef.